### ISSUE TITLE
Error when displaying dataFrame with binary(16) type

### ISSUE THREAD

**Issue by IzakMarais** [vertica/VerticaPy#154]
I create a vDataFrame from an existing table that has a binary(16) type for one of the columns. When I try to display the contents of the vDataFrame, I get a stack trace:

```
t:\repos\notebooks\.venv\lib\site-packages\vertica_python\vertica\cursor.py in _execute_simple_query(self, query)
    604         self._message = self.connection.read_message()
    605         if isinstance(self._message, messages.ErrorResponse):
--> 606             raise errors.QueryError.from_error_response(self._message, query)
    607         elif isinstance(self._message, messages.RowDescription):
    608             self.description = [Column(fd, self.unicode_error) for fd in self._message.fields]

QueryError: Severity: ERROR, Message: Function ST_AsText(binary) does not exist, or permission is denied for ST_AsText(binary), Sqlstate: 42883, Hint: No function matches the given name and argument types. 
You may need to add explicit type casts, Routine: throwErrorFunctionDoesNotMatch, 
File: /data/qb_workspaces/jenkins2/ReleaseBuilds/Hammermill/REL-10_1_1-x_hammermill/build/vertica/Parser/ParseFunc.cpp,
Line: 3553, Error Code: 3457, SQL: 'SELECT "id" AS "id", ST_AsText("uuid") AS "uuid",...
```
In the above stack trace the `uuid` column is of type `binary(16)`.

I did not get the same error when using SQL magic to display the tableSample.

So:
```
df = verticapy.vDataFrame("Table", cursor=cur)
df
```
would fail, while
```
%sql SELECT * from Table
```
would not.

**Comment by oualib** [comment]
Thanks for reporting this issue, it will be fixed in v0.7.0.

**Comment by IzakMarais** [comment]
Thanks for the quick feedback.
